### ISSUE TITLE
operator: Missing operator container images on docker hub

### ISSUE THREAD

**Issue by periklis** [grafana/loki#5617]
**Describe the bug**
Starting with merging the operator subproject into grafana/loki #4945 the SIG decided to publish operator container images on two registries in parallel:
- operator containers images for amd64 and arm64 on https://hub.docker.com/r/grafana/loki-operator
- operator, bundle and broker container images for Red Hat members (using them in OpenShift Logging) on https://quay.io/organization/openshift-logging

Due to a time conflict, the creation of the docker hub repo took some time. However, since it is available, we miss the images being published there on every merge into main.

**Expected behavior**
The images are available on docker hub as described above.

**Comment by Red-GV** [comment]
I believe this was fixed by: https://github.com/grafana/loki/pull/6086 and https://github.com/grafana/loki/pull/6599
